src/components/views/Experiment.js

```diff
@@ -74,7 +74,6 @@ export default class extends React.Component {
         if (this.props.description) {
             maybeDescription = (
                 <section id="experiment-description">
-                    <h4>Description</h4>
```

Comment:
Even these headings aren't not shown, they're semantically relevant and should be useful to screen readers, search engines, etc.